### PR TITLE
447 bump node to version 18

### PR DESCRIPTION
**Pull request by DmitryLukyanov**
## Description
Migration to node 18

## Type of changes
Configuration changes.

## Checklist

- [X] I have read [CONTRIBUTING](https://github.com/epam/miew/blob/master/CONTRIBUTING.md) and [CODE_OF_CONDUCT](https://github.com/epam/miew/blob/master/CODE_OF_CONDUCT.md) guides.
- [X] I have followed the code style of this project.
- [X] I have run `yarn run ci`: lint and tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation / The changes do not need docs update.

### CODE REVIEW

**Review by paulsmirnov — CHANGES_REQUESTED** [review]
The changes seem reasonable. Thank you!

I've noticed one issue with the `packages/app`: images are missing in the Mode/Color dropdowns and in the main menu. The same works fine in the `main` branch. Perhaps, is it something to do with the new webpack. Could you please confirm that it is not an issue with only my setup (I've checked twice)?

Steps to reproduce:
```bash
nvm use 18  # or "nvm use 14" for the main branch
yarn install
cd packages/lib
yarn run ci
cd ../app
yarn start
```

Actual vs expected:
shot-20230813-112843 shot-20230813-114334

**Comment by DmitryLukyanov** [comment]
> The changes seem reasonable. Thank you!
> 
> I've noticed one issue with the `packages/app`: images are missing in the Mode/Color dropdowns and in the main menu. The same works fine in the `main` branch. Perhaps, is it something to do with the new webpack. Could you please confirm that it is not an issue with only my setup (I've checked twice)?
> 
> Steps to reproduce:
> 
> ```shell
> nvm use 18  # or "nvm use 14" for the main branch
> yarn install
> cd packages/lib
> yarn run ci
> cd ../app
> yarn start
> ```
> 
> Actual vs expected: shot-20230813-112843 shot-20230813-114334

yeah, I see the same, required links on images gives 404. I will look at it.

Btw, do we have any test on this?

**Comment by paulsmirnov** [comment]
> Btw, do we have any test on this?

None that I'm aware of. We'll need to decide on the implementation because it doesn't look obvious. HTML snapshots are not enough while 3D screenshots can be different depending on software and hardware.

**Comment by DmitryLukyanov** [comment]
> > Btw, do we have any test on this?
> 
> None that I'm aware of. We'll need to decide on the implementation because it doesn't look obvious. HTML snapshots are not enough while 3D screenshots can be different depending on software and hardware.

we can check that at least one picture link doesn't return 404 :)

**Review by NataliaDSmirnova — DISMISSED** [review]
Awesome job!

**Comment by NataliaDSmirnova** [comment]
@DmitryLukyanov, 
It seems the problem with images and `url-loader` limit question may be linked. The reason `url-loader` is deprecated in Webpack5.
See https://v4.webpack.js.org/loaders/url-loader/ and https://webpack.js.org/guides/asset-modules/ 

The information above is provided by @paulsmirnov and @AlinaFirsova. I hope it helps 🙂

File: .appveyor.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "{build}"
 environment:
   matrix:
-    - nodejs_version: "14"
+    - nodejs_version: "18"
```
[the remark this answers is not in the document]
Comment (DmitryLukyanov) [reply .appveyor.yml:4]: It looks like it's related to lerna that we're going to remove in a parallel ticket

**Review by NataliaDSmirnova — APPROVED** [review]
Great! I've checked:
- lib: yarn build,  yarn start -> demo works (shortcuts also), examples work
- app: yarn build, yarn start -> app works (shortcuts also)

Big thanks ♥

**Review by paulsmirnov — APPROVED** [review]
Awesome! The steps that failed previously work for me now. I see the images. I hope we also understand the cause :)

**Comment by paulsmirnov** [comment]
Are we going to get rid of the deprecated modules later in a separate task?

**Pull request merged**